pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -3653,9 +3654,113 @@ func Test_server_MultiInsert(t *testing.T) {
 		beforeFunc func(args)
 		afterFunc  func(args)
 	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// common variables for test
+	const (
+		name      = "vald-agent-ngt-1" // agent name
+		id        = "uuid-1"           // insert request id
+		intVecDim = 3                  // int vector dimension
+		f32VecDim = 3                  // float32 vector dimension
+	)
+	var (
+		ip     = net.LoadLocalIP()        // agent ip address
+		intVec = []float32{1, 2, 3}       // int vector of the insert request
+		f32Vec = []float32{1.5, 2.3, 3.6} // float32 vector of the insert request
```

Comment:
could you please use generate vector function?

Reply:
fixed by removing this variable and use `genF32Req` in test case.